go.mod

```diff
@@ -169,6 +169,7 @@ require (
 replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/cometbft/cometbft => github.com/yihuang/cometbft v0.38.0-alpha.1.0.20230621081303-4a1c1df1c182
```

Comment:
https://github.com/cometbft/cometbft/pull/1011

Reply:
Would honestly prefer to not pin against a commit hash and instead keep this PR open (and blocked) until merged and released in CometBFT. Otherwise LGTM!

Reply:
converted to draft again ;D

Reply:
Yeah, personally I would prefer for them to include it in their next RC 🙏